### DISCUSSION TITLE
Collecting what you need is better and more convenient than cutting out what you don’t need.

### DISCUSSION THREAD

**Discussion by AndreyGasik** [ChrisTitusTech/winutil#2980] [Ideas]
My suggestion is not to disassemble what you have collected, but to collect what you need. You can download an assembly script from the UUPDUMP website, in this script you can select which components are needed, and even edge is simply not installed in the ISO, the system turns out clean and without garbage. Then you can apply microwin tweaks and get ideal Windows ISO with much more flexible selection conditions. I don't suggest taking their script, you can just create something of your own. After all, you must admit that assembling the necessary parts is better than disassembling and cutting out the unnecessary.

**Comment by Pancacasaurus** [comment]
Can you elaborate more, or are there any guides on this?

**Comment by AndreyGasik** [comment]
Sure, just go to [uupdump](https://uupdump.net/) choose your build, i choose latest 21600.2161, choose languange, press next, choose edition, press next, press button  download and convert iso, include updates, integrate net, run component cleanup NO need,  press create download page. Now you will recieve script in zip file. Unzip it. In folder now have CustomAppsList.txt and ConvertConfig.ini
Open CustomAppsList.txt in notepad, just delete # symbol what soft you need, where have # not install.after just save it. in file ConvertConfig.ini Just NEED check CustomList =1, and here you can delete Edge if no need SkipEdge=1 . Its all, after this run from admin uup_download_windows.cmd and wait your clean iso.

**Comment by AndreyGasik** [comment]
Its all download from microsoft, you can see all process, so just try, its very easy.
